pubtools/pulplib/_impl/model/repository/base.py

```diff
@@ -159,6 +159,20 @@ class Repository(PulpObject):
     _client = attr.ib(default=None, init=False, repr=False, cmp=False, hash=False)
     # hidden attribute for client attached to this object
 
+    @distributors.validator
+    def check_repo_id(self, _, value):
```

Comment:
Should also be named with _ to make it private API, this method isn't meant to be invoked directly by users.

```suggestion
    def _check_repo_id(self, _, value):
```

Reply:
I'd have committed this fix myself but it seems like I don't have permission (is the checkbox ticked to allow maintainers to edit this pull request?)

Reply:
Yes, it is checked.